app/hyperterm.js

```diff
@@ -373,8 +367,8 @@ export default class HyperTerm extends Component {
       keys.bind('command+shift+]', this.moveRight);
       keys.bind('command+alt+left', this.moveLeft);
       keys.bind('command+alt+right', this.moveRight);
-      keys.bind('command+=', this.increaseFontSize);
-      keys.bind('command+-', this.decreaseFontSize);
+      keys.bind('command+plus', this.increaseFontSize);
+      keys.bind('command+minus', this.decreaseFontSize);
```

Comment:
you're quite right (this is probably why the listener was being called twice) fixed